lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/ninoCri/ContractTest.java

```diff
@@ -0,0 +1,990 @@
+             "type": [
+               "VerifiableCredential",
+               "IdentityCheckCredential"
+             ]
+           },
+           "jti":"dummyJti"
+         }
+    """;
+    // If we generate the signature in code it will be different each time, so we need to generate a
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String VALID_NINO_IDENTITY_CHECK_VC_SIGNATURE =
+            "WHEg4EooDiCdT676GPVE4l4z5YY-Tvl3FTY4bYI_nCzx8mqrg2rTYJUM4SG8azbyBjdSG8UQFyfAuLPeYlMTwg";
+
+    private static final String FAILED_NINO_IDENTITY_CHECK_VC_BODY =
+            """
+            {
+                "sub": "test-subject",
+                "iss": "dummyNinoComponentId",
+                "nbf": 4070908800,
+                "exp": 4070909400,
+                "vc": {
+                 "evidence": [
+                   {
+                     "failedCheckDetails": [
+                       {
+                         "checkMethod": "data"
+                       }
+                     ],
+                     "validityScore": 0,
+                     "strengthScore": 2,
+                     "ci": [
+                        "D02"
+                     ],
+                     "type": "IdentityCheck",
+                     "txn": "dummyTxn"
+                   }
+                 ],
+                 "credentialSubject": {
+                   "socialSecurityRecord": [
+                     {
+                       "personalNumber": "AA000003D"
+                     }
+                   ],
+                   "name": [
+                     {
+                       "nameParts": [
+                         {
+                           "type": "GivenName",
+                           "value": "Kenneth"
+                         },
+                         {
+                           "type": "FamilyName",
+                           "value": "Decerqueira"
+                         }
+                       ]
+                     }
+                   ],
+                   "birthDate": [
+                     {
+                       "value": "1965-07-08"
+                     }
+                   ]
+                 },
+                 "type": [
+                   "VerifiableCredential",
+                   "IdentityCheckCredential"
+                 ]
+                },
+                "jti":"dummyJti"
+                }
+            """;
+    // If we generate the signature in code it will be different each time, so we need to generate a
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String FAILED_NINO_IDENTITY_CHECK_VC_SIGNATURE =
+            "GdAM1VaIj5fPZD-8mosORKfbqazER2xNzS2wJ-3ksq2HsCLujvK6kTSjER1OxMHTXH4T8wEROVbGIVeaK9vY9A";
+
+    private static final String VALID_NINO_VC_BODY =
```

Comment:
Not a big deal but I'd maybe name this `VALID_NINO_RECORD_CHECK_VC` just to clearly distinguish the two